frontend/admin/src/js/components/user/GeneralInformationTab/AddToPoolDialog.tsx

```diff
@@ -89,6 +139,20 @@ export const AddToPoolDialog: React.FC<AddToPoolDialogProps> = ({
   };
   const { handleSubmit } = methods;
 
+  const poolOptions = pools
+    // .filter((pool) => !currentPools.includes(pool.id))
```

Comment:
could you remove commented code ?

Reply:
😬 That wasn't supposed to be committed. Great catch!

Reply:
ooh .. thats why I got the pools I have already been in .. 😆

Reply:
Yeah, I commented it out to force and error while testing by attempting to add a user to a pool they already belonged to 😜